recipes/libselinux/all/patches/0001-fix-fno-common-3.0.patch

```diff
@@ -0,0 +1,510 @@
+libsepol: fix CIL_KEY_* build errors with -fno-common
+
+GCC 10 comes with -fno-common enabled by default - fix the CIL_KEY_*
+global variables to be defined only once in cil.c and declared in the
+header file correctly with the 'extern' keyword, so that other units
+including the file don't generate duplicate definitions.
+
+see https://github.com/SELinuxProject/selinux/commit/a96e8c59ecac84096d870b42701a504791a8cc8c
```

Comment:
This is really border line for #3903 ... I not a fan of backporting patches but CCI allows expanding build support... blah

My personal preference is setting compilers limits... who's going to use the old version on new platforms?